gulpfile.babel.ts

```diff
@@ -189,6 +189,7 @@ async function browserifyTask(options: BrowserifyTaskOptions): Promise<void> {
   })
     .transform(
       babelify.configure({
+        presets: ['@babel/preset-env'],
```

Comment:
Why is this specified here?

Reply:
From https://github.com/babel/babelify#babelify-

> As of [Babel 6.0.0](http://babeljs.io/blog/2015/10/29/6.0.0/) there are no plugins included by default. For babelify to be useful, you must also include some [presets](http://babeljs.io/docs/plugins/#presets) and/or [plugins](http://babeljs.io/docs/plugins/#transform).

Reply:
This was indeed a spurious bit of config. We chatted about it being unnecessary offline.